pkgs/by-name/sh/shoutrrr/package.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
```

Comment:
have you passed this new file through `nixfmt-rfc-style` ?

Reply:
sorry I wasn't aware of that nixfmt version